.env.test

```diff
@@ -5,3 +5,10 @@
 # To adapt to your local setup, copy this file to `.env.test.local` to make changes
 #
 DATABASE_URL=postgresql://postgres@localhost/laa-criminal-applications-datastore-test
+
+# ElasticMQ endpoint, only used for local development/tests
+# In k8s cloud environments, real AWS SQS is used instead
+LOCAL_ELASTICMQ_URL=http://localhost:9324
+
+# MAAT SQS queue URL (this will point to AWS on k8s)
+SQS_MAAT_QUEUE_URL="${LOCAL_ELASTICMQ_URL}/test_submitted_applications_for_maat"
```

Comment:
So I've noticed other LAA services using queues for integration with other systems are super explicit about what is the queue for, having their name in the queue itself. For instance these queues from CCCD: `cccd-claims-for-ccr` and `cccd-claims-for-cclf` (and then the corresponding dead letter queues are the same with the `-dlq` suffix).
I thought this might be a good idea.

I suppose we can still give it an internal (in our codebase) meaningful name only for us, and then have the actual SQS named with as much detail as possible.

Reply:
👍